frontend/pages/register.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <h1>Register receiving address</h1>
+    <p v-if="$fetchState.pending">
+      Loading...
+    </p>
+    <p v-else-if="$fetchState.error" class="error">
+      An error occurs...
+    </p>
+    <div v-else-if="valid">
```

Comment:
Can use template for just wrapper without extra div
```suggestion
    <template v-else-if="valid">
```